main.tf

```diff
@@ -146,6 +147,35 @@ resource "kubernetes_deployment" "wandb" {
             value = var.cloud_monitoring_connection_string
           }
 
+          env {
+            name = "WEAVE_SERVICE"
+            value = var.weave_enabled ? "${kubernetes_service.weave.0.metadata.0.name}:9994" : ""
+          }
+
+          env {
+            name = "GORILLA_STATSD_HOST"
+            value_from {
+              field_ref {
+                field_path = "status.hostIP"
+              }
+            }
+          }
+
+          env {
+            name = "GORILLA_STATSD_PORT"
+            value = "8125"
+          }
```

Comment:
I think these need to be opt-in.  Lot's of clusters won't have a DD agent accepting StatsD metrics.

Reply:
hmm, lemme just remove these two then, it can be passed via other_wandb_env